docs/src/main/asciidoc/cli-tooling.adoc

```diff
@@ -20,114 +20,151 @@ The Quarkus CLI is currently available as a jar installable using https://jbang.
 On Linux, macOS, and Windows (using WSL or bash compatible shell like cygwin or mingw)
 [source,bash]
 ----
-curl -Ls https://sh.jbang.dev | bash -s - app install --force qs@quarkusio
+curl -Ls https://sh.jbang.dev | bash -s - app install --force quarkus@quarkusio
```

Comment:
```
jbang app install --force quarkus@quarkusio
[jbang] [ERROR] No alias found with name 'quarkus'
[jbang] Run with --verbose for more details
```

Maybe worth adding also `jbang app install --force quarkus@quarkusio` for people who have jbang already installed.